multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DetermineServiceCreateUpdateServiceActionsStep.java

```diff
@@ -40,10 +43,16 @@
 import com.sap.cloudfoundry.client.facade.domain.CloudServiceKey;
 import com.sap.cloudfoundry.client.facade.domain.ServiceOperation;
 
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+
 @Named("determineServiceCreateUpdateActionsStep")
 @Scope(BeanDefinition.SCOPE_PROTOTYPE)
 public class DetermineServiceCreateUpdateServiceActionsStep extends SyncFlowableStep {
 
+    @Inject
+    private ArchiveEntryExtractor archiveEntryExtractor;
```

Comment:
Can you make it with contructor injection

Reply:
done